SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKDefaultLaunchConfigGenerator.cpp

```diff
@@ -211,7 +211,8 @@ bool ConvertToClassicConfig(const FString& LaunchConfigPath, const FSpatialLaunc
 
 	if (ExitCode != SpatialGDKServicesConstants::ExitCodeSuccess)
 	{
-		UE_LOG(LogSpatialGDKDefaultLaunchConfigGenerator, Error,
+		// TODO: Change back to Error when v15 is released so we aren't getting unnecessary error spam.
```

Comment:
Worth a UNR ticket?

Reply:
Nah. In this case it's to allow the tests to pass. This won't go into master ever as it'll be addressed before v15 is merged :D